README.md

```diff
@@ -68,43 +68,50 @@ These xRunBooks are included in every install.  Use them as is, or make a copy t
+| Kubernetes | [K8S Pod Stuck In CrashLoopBack State](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/Kubernetes/K8S_Pod_Stuck_In_CrashLoopBack_State.ipynb) | [Open in browser](http://127.0.0.1:8888/lab/tree/K8S_Pod_Stuck_In_CrashLoopBack_State.ipynb)
+| Kubernetes | [K8S Pod Stuck In ImagePullBackOff State](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/Kubernetes/K8S_Pod_Stuck_In_ImagePullBackOff_State.ipynb) | [Open in browser](http://127.0.0.1:8888/lab/tree/K8S_Pod_Stuck_In_ImagePullBackOff_State.ipynb)
+| Kubernetes | [K8S Pod Stuck In Terminating State](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/Kubernetes/K8S_Pod_Stuck_In_Terminating_State.ipynb) | [Open in browser](http://127.0.0.1:8888/lab/tree/K8S_Pod_Stuck_In_Terminating_State.ipynb)
+| Kubernetes | [Rollback k8s Deployment and Update Jira](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/Kubernetes/Rollback-k8s-Deployment-and-Update-Jira.ipynb) | [Open in browser](http://127.0.0.1:8888/lab/tree/Rollback-k8s-Deployment-and-Update-Jira.ipynb)
+| Mongo | [MongoDB Server Connectivity](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/Mongo/MongoDB_Server_Connectivity.ipynb) | [Open in browser](http://127.0.0.1:8888/lab/tree/MongoDB_Server_Connectivity.ipynb)
+| MySQL | [MySQL Long Run Query](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/MySQL/MySQL_Long_Run_Query.ipynb) | [Open in browser](http://127.0.0.1:8888/lab/tree/MySQL_Long_Run_Query.ipynb)
+| Postgresql | [Display postgresql long running](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/Postgresql/Display-postgresql-long-running.ipynb) | [Open in browser](http://127.0.0.1:8888/lab/tree/Display-postgresql-long-running.ipynb)
 
-### How to Contribute?
-
-1. Work with us on any of our [open issues](https://github.com/unskript/awesome-cloudops-automation/issues).
-2. Create a new Action. Read the [Action developer guide](https://github.com/unskript/Awesome-CloudOps-Automation/blob/master/.github/DEVELOPERGUIDE.md) for more details.
-3. Combine Actions into a xRunBook.  Your xRunBook will be featured on the ReadMe.  [What's a xRunBook?](https://github.com/unskript/Awesome-CloudOps-Automation/blob/master/.github/Anatomy-of-Runbook.md)
 
 
-
-### Contribution Tips:
-1. Use the [Docker environment](#get-started-with-docker), or our free [cloud sandbox](https://us.app.unskript.io/profiles/6c38d3da1cde7b3c0623d138f525a5508a3260c8) for testing your contribution.
-2. Join our [Slack Community](https://communityinviter.com/apps/cloud-ops-community/awesome-cloud-automation) so you can present yourself and get support.
+### How to Contribute?
```

Comment:
Are we moving this section (How to Contribute) out to somewhere else? Should we add a link instead?

Reply:
The readme.md is built from the readme_template.md.  so it doesn't matter what is in the current readme, the action will overwrite the file.

These lines are in the tenmplate file, so will be in the readme on completion of the action